### PR TITLE
Update used go version to 1.19

### PR DESCRIPTION
**Pull request by johscheuer**

### CODE REVIEW

File: docker/centos7/setup.sh [hunk 1]
```diff
@@ -1,623 +0,0 @@
-#!/usr/bin/env bash
```
Comment (johscheuer) [inline docker/centos7/setup.sh:1]: I couldn't find a place where this script is used, so I think we can remove it.

Comment (ammolitor) [reply docker/centos7/setup.sh:1]: This is correct, is is not used in the repo. It is used to setup the release EC2 instance, and _should_ align with the Dockerfile. It should stay.

**Review by ammolitor — CHANGES_REQUESTED** [review]
Please add back the `setup.sh` file, otherwise LGTM.

File: docker/centos7/setup.sh (hunk 1 above)
Comment (johscheuer) [reply docker/centos7/setup.sh:1]: Good to know, let me add a comment to that script. I wonder if using the script inside the Docker image is a better idea to keep those files aligned e.g. currently they already diverge by installing go 1.18 and go 1.17.

**Review by saintstack — APPROVED** [review]
LGTM. +1 if you've tried it and it all installed.

Unifying the install_golang.sh scripts at one so just one w/ args w/ be a PITA?

**Review by ammolitor — CHANGES_REQUESTED** [review]
Please put the commands in the `Dockerfile`.

**Comment by johscheuer** [comment]
> Please put the commands in the `Dockerfile`.

Done :)

**Comment by saintstack** [comment]
Why is that @ammolitor  ? You don't like having to look elsewhere? Just out of interest...

**Comment by saintstack** [comment]
+1 again from me.

**Comment by ammolitor** [comment]
> Why is that @ammolitor ? You don't like having to look elsewhere? Just out of interest...

If we want to redesign our Dockerfile build process, I'm open to that... in the right context. For me, this should be a surgical change (e.g. only change what is necessary) to move the golang version forward.

**Comment by johscheuer** [comment]
> > Why is that @ammolitor ? You don't like having to look elsewhere? Just out of interest...
> 
> If we want to redesign our Dockerfile build process, I'm open to that... in the right context. For me, this should be a surgical change (e.g. only change what is necessary) to move the golang version forward.

I agree with that 👍

**Pull request merged**